README.md

```diff
@@ -47,4 +89,61 @@ The project is current divided into three repositories handling complementary as
 - `falsepos-...` files are cases any filter should catch
 - `truepos...` files are cases of changes we care about
 
-Filters will need to be tested (and possibly trained) against real datasets at some point
+This is a small but illustrative sample. Many more samples will be made
+available as soon as possible.
+
+### Schema
+
+This describes the schema of the SQL databases shared by the Rails app in
+web-monitoring-db and the Python processing bakend in web-monitoring-processing.
+Review the Definition of Terms section above, which corresponds to these tables.
+
+Every table includes:
+
+* uuid: UUID4 unique indentifier
+* created_at: internal detail of the database
+* updated_at: internal detail of the database
+
+in addition to the table-specific fields listed below.
+
+#### Pages
+
+* url: URL, which may be updated over time if a page is moved
+* title: `<title>` tag
+* agency: Governemnt agency
+* site: Goverment site (?)
+
+#### Versions
+
+* page_uuid: reference to a Page
+* capture_time: when this snapshot of the Page was acquired
+* uri: path to stored (HTML) data; could be a filepath, S3 bucket, etc.
+* version_hash: sha256 hash of stored data
+* source_type: name of source (such as 'Internet Archive')
+* source_metadata: JSON blob of extra info particular to the source
+
+#### Changes
+
+* uuid_from: reference to the "before" Version
+* uuid_to: reference to the "after" Version
+* priority: a number between 0 and 1 where 1 is high priority
+* current_annotation: for convenience, a reduction of one or more submitted Annotations
+
+####  Diffs
+
+* change_uuid: reference to a Change that this Diff represents
+* uri: path to stored diff data; could be a filepath, S3 bucket, etc.
+* diff_hash: sha256 has of stored diff data
+* source_type: name of diffing utility (such as 'PageFreezer')
+* source_metadata: JSON blob of extra info particular to the source
+
+#### Annotations
+
+* change_uuid: reference to a Change that this Annotation characterizes
+* annotation: JSON blob
+* author: user id
+
+(This summary omits Users and Invitations, which are implemented in the Rails
+app.)
+
+For more details see the Python implementation [here](https://github.com/edgi-govdata-archiving/web-monitoring-processing/blob/master/web_monitoring/db.py) and the Ruby implementation (currently in progress) [here](https://github.com/edgi-govdata-archiving/web-monitoring-db/pull/15).
```

Comment:
This is a bit picky, but for clarity I'm generally against non-descriptive links (e.g. "here"), can we just have the "Python implementation / Ruby implementation" as the links?